packages/lablqml/lablqml.0.6.2/opam

```diff
@@ -0,0 +1,41 @@
+opam-version: "2.0"
+maintainer:   "kakadu.hafanana@gmail.com"
+authors:      "kakadu.hafanana@gmail.com"
+homepage:     "http://kakadu.github.io/lablqt/"
+bug-reports:  "https://github.com/kakadu/lablqml/issues"
+dev-repo: "git+https://github.com/Kakadu/lablqml.git"
+tags: [ "gui" "ui" "qt" ]
+
+build: [
+  ["sh" "-exc" "PATH=/usr/lib64/qt5/bin:/usr/lib/qt5/bin:$PATH ./configure"] {os-distribution = "alpine" | os-distribution = "centos" | os-distribution = "fedora" }
+  ["sh" "-exc" "PATH=/usr/lib64/qt5/bin:/usr/lib/qt5/bin:$PATH make"]        {os-distribution = "alpine" | os-distribution = "centos" | os-distribution = "fedora" }
+  ["sh" "-exc" "PATH=/usr/lib64/qt5/bin:/usr/lib/qt5/bin:$PATH make demos"]  
+    { (os-distribution = "alpine"  | os-distribution = "centos" | os-distribution = "fedora") & with-test }
+  
+  ["./configure"]            { os-distribution != "alpine" & os-distribution != "centos" & os-distribution != "fedora" }
+  [make]                     { os-distribution != "alpine" & os-distribution != "centos" & os-distribution != "fedora" }
```

Comment:
I can't figure out how you use dune. If you're only using dune you shouldn't have to use make and also using `dune build` without `-p` is to be proscribed

Reply:
At the moment I have simple Makefile that executes dune. './configure` at the moment links hello world Qt application. It could be ported to dune using `configurator` I suppose. But later.

Reply:
I'm not talking about the configure script, this one can stay, I'm talking about the make call (which calls `dune build` without any `-p` or `--profile=release`)

Reply:
It should be better now.